source/renderer/app/utils/storesUtils.js

```diff
@@ -17,7 +17,7 @@ export const getRequestKeys = (
   };
 };
 
-export const requestGetter = (req: Request, systemValue: string) => {
+export const requestGetter = (req: Request, systemValue: any) => {
```

Comment:
@daniloprates Can you check `systemValue` type and change it from `any` if is easy?

Reply:
Needs to be `any`, really, because you can really pass any type of variable, depending on your Request

Reply:
@daniloprates 👍